test/integration/proportion-boolean/patients/patient-ipp-bundle.json

```diff
@@ -0,0 +1,151 @@
+            "value": "test-patient-bccc8f75-cf49-4f9d-be9f-b5f63c247e32"
+          }
+        ],
+        "name": [
+          {
+            "family": "Iminipp",
+            "given": [
+              "Isaac"
+            ]
+          }
+        ],
+        "gender": "female",
+        "birthDate": "2001-01-01"
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Patient/bccc8f75-cf49-4f9d-be9f-b5f63c247e32"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "Procedure",
+        "id": "95d733ba-da65-4263-bd8c-fdbcd1508711",
+        "code": {
+          "coding": [
+            {
+              "system": "exampleSystem",
+              "version": "exampleVersion",
+              "code": "exampleCode"
+            }
+          ]
+        },
+        "status": "completed",
+        "subject": {
+          "reference": "Patient/bccc8f75-cf49-4f9d-be9f-b5f63c247e32"
+        },
+        "performedPeriod": {
+          "start": "2022-01-03T16:19:20.391Z",
+          "end": "2022-01-04T16:19:20.391Z"
+        }
+      },
+      "request": {
+        "method": "PUT",
+        "url": "Procedure/95d733ba-da65-4263-bd8c-fdbcd1508711"
+      }
+    },
+    {
+      "resource": {
+        "resourceType": "MeasureReport",
+        "id": "d8f7acb4-bf44-460d-9677-c79b007f3941",
+        "measure": "http://example.com/Measure/measure-ProportionBooleanTest",
+        "period": {
+          "start": "2022-01-01T05:00:00.000Z",
+          "end": "2022-12-31T05:00:00.000Z"
+        },
+        "status": "complete",
+        "type": "individual",
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/test-case-cqfm"
+          ]
+        },
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-inputParameters",
+            "valueReference": {
+              "reference": "#10afeadf-27d6-42eb-b4e9-3df0c8920d42"
+            }
+          }
+        ],
+        "modifierExtension": [
+          {
+            "url": "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-isTestCase",
+            "valueBoolean": true
+          }
+        ],
+        "contained": [
+          {
+            "resourceType": "Parameters",
+            "id": "10afeadf-27d6-42eb-b4e9-3df0c8920d42",
+            "parameter": [
+              {
+                "name": "subject",
+                "valueString": "bccc8f75-cf49-4f9d-be9f-b5f63c247e32"
+              }
+            ]
+          }
+        ],
+        "group": [
+          {
+            "population": [
+              {
+                "code": {
+                  "coding": [
+                    {
+                      "system": "http://terminology.hl7.org/CodeSystem/measure-population",
+                      "code": "initial-population"
+                    }
+                  ]
+                },
+                "count": 0
```

Comment:
It doesn't _really_ matter for the sake of this integration test, but maybe in the future if we ever want to do anything with the test case measure report, we should properly reflect it in fqm-testify.

AKA can you actually select the desired populations in testify before exporting the patient?